utilities/db_util.py

```diff
@@ -0,0 +1,41 @@
+from dotenv import dotenv_values
+import mysql.connector
+
+def connect_to_db(db: str=""):
+  config = dotenv_values(".env")
+  try:
+    if len(db)==0:
+      mydb = mysql.connector.connect(
+        host="localhost",
+        user=config['USER'],
+        password=config['PASSWORD'],
+      )
+    else:
+      mydb = mysql.connector.connect(
+        host="localhost",
+        user=config['USER'],
+        password=config['PASSWORD'],
+        database=db,
+      )
```

Comment:
Perhaps something like this to not duplicate code (since only difference in if/else is `database` param):
```
db_config = {
    'host': 'localhost',
    'user': config['USER'],
    'password': config['PASSWORD']
}

if db:
    db_config['database'] = db

mydb = mysql.connector.connect(**db_config)
```